src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj

```diff
@@ -413,18 +446,35 @@ This package is an internal implementation of the .NET Core SDK and is not meant
           Condition="'$(CrossCompileDirectory)' == ''">$(RuntimePackageRoot)runtimes\$(RuntimeIdentifier)\native\$(LibPrefix)clrjit$(LibExtension)</CoreCLRJitPath>
       <CoreCLRJitPath
           Condition="'$(CrossCompileDirectory)' != ''">$(RuntimePackageRoot)runtimes\$(CrossCompileDirectory)\native\$(LibPrefix)clrjit$(LibExtension)</CoreCLRJitPath>
+
+      <Crossgen2TargetOs>$(TargetOsName)</Crossgen2TargetOs>
+      <Crossgen2TargetOs Condition="'$(TargetOsName)' == 'win'">windows</Crossgen2TargetOs>
+      <Crossgen2TargetOs Condition="'$(TargetOsName)' == 'linux-musl'">linux</Crossgen2TargetOs>
+      <Crossgen2Args>--targetarch:$(TargetArchitecture)</Crossgen2Args>
+      <Crossgen2Args>$(Crossgen2Args) --targetos:$(Crossgen2TargetOs)</Crossgen2Args>
+      <Crossgen2Args>$(Crossgen2Args) -O</Crossgen2Args>
+      <Crossgen2Args>$(Crossgen2Args) @&quot;$(CrossgenToolDir)PlatformAssembliesPathsCrossgen2.rsp&quot;</Crossgen2Args>
+      <Crossgen2Args Condition="Exists('$(RuntimePackageRoot)tools\StandardOptimizationData.mibc')">$(Crossgen2Args) &quot;-m:$(RuntimePackageRoot)tools\StandardOptimizationData.mibc&quot;</Crossgen2Args>
```

Comment:
What is the StandardOptimizationData.mibc file and does it have anything to do with ASP.NET Core scenarios❔ I as expecting to check in PGO data (if that's what this file is about) specific to this repo.

Reply:
StandardOptimizationData is the training that is produced from the optimization repo. It involves a variety of scenarios, such as TechEmpower plaintext mvc, a simple helloworld, various sdk activities. There are plans to expand the coverage of asp.net scenarios, by adding test scenarios such as Orchard as well. The end result is that StandardOptimizationData holds optimization data relevant to this repo. If you wish to build up a parallel infrastructure, it is in theory possible, but my recommendation is to just piggy back on this data.